### PR TITLE
final commit

### PR DESCRIPTION
**Pull request by metrio**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -68,7 +68,43 @@ function toggleColor(element) {
   }
 }
 
+const h1Header = document.querySelector("#header")
+
+h1Header.addEventListener("click", function() {
+  toggleColor(h1Header)
+})
+
```
Comment (BrendaMichelle) [inline index.js:76]: Looks good!

File: index.js [hunk 2]
```diff
@@ -68,7 +68,43 @@ function toggleColor(element) {
...
+
 /***** Deliverable 2 *****/
+const likeBtn = document.querySelector('.like-button')
+const pLike = document.querySelector('.likes')
+
+likeBtn.addEventListener('click', function(){
+  const likes = parseInt(pLike.textContent) + 1 + " Likes"
+  pLike.textContent = likes
+})
```
Comment (BrendaMichelle) [inline index.js:86]: Deliverable 2 looks good! The key here is that you have to read some information from the DOM to get the current number of likes, increment it, and then write it back to the DOM. 💯

File: index.js [hunk 3]
```diff
@@ -68,7 +68,43 @@ function toggleColor(element) {
...
+  const link = event.target.link.value
+  const description = event.target.description.value
+
+  const goalId = document.querySelector('#goals').lastChild.dataset.id
+  const newId = parseINt(goalId) + 1
+
+  const newGoal = {
+    id: newId,
+    link: link,
+    description: description
```
Comment (BrendaMichelle) [inline index.js:106]: Nice work! You could shorten your code by declaring the values in the object directly:

```Javascript
const player = {
    id: parseInt(goalId) + 1,
    number: event.target.link.value,
    name: event.target.description.value
}
```

What if the ul#goals doesn't have any children li elements? Then `goaldId` would be undefined and the ID incrementation wouldn't work properly. You could add a conditional to check for that edge case. You could also pull the latest ID from the `goals` array in the `player` object in `data.js`.  Also, be sure to test your code thoroughly!  The typo in `parseINt` will result in a 'not a function' error.

**Review by BrendaMichelle — COMMENTED** [review]
Great work, keep it up!
